[2025-06-19T13:21:07.825+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: week_3_weather_api_dag_v2.load_data_to_postgres manual__2025-06-19T13:20:44.027514+00:00 [queued]>
[2025-06-19T13:21:07.835+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: week_3_weather_api_dag_v2.load_data_to_postgres manual__2025-06-19T13:20:44.027514+00:00 [queued]>
[2025-06-19T13:21:07.836+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-06-19T13:21:07.848+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): load_data_to_postgres> on 2025-06-19 13:20:44.027514+00:00
[2025-06-19T13:21:07.865+0000] {standard_task_runner.py:60} INFO - Started process 380 to run task
[2025-06-19T13:21:07.872+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'week_3_weather_api_dag_v2', 'load_data_to_postgres', 'manual__2025-06-19T13:20:44.027514+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/week_3_weather_api_dag_v2.py', '--cfg-path', '/tmp/tmp0ayaplhc']
[2025-06-19T13:21:07.874+0000] {standard_task_runner.py:88} INFO - Job 88: Subtask load_data_to_postgres
[2025-06-19T13:21:07.929+0000] {task_command.py:423} INFO - Running <TaskInstance: week_3_weather_api_dag_v2.load_data_to_postgres manual__2025-06-19T13:20:44.027514+00:00 [running]> on host 5513fff5175e
[2025-06-19T13:21:08.017+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='supitchaya.p99@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='week_3_weather_api_dag_v2' AIRFLOW_CTX_TASK_ID='load_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-06-19T13:20:44.027514+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-19T13:20:44.027514+00:00'
[2025-06-19T13:21:08.027+0000] {warnings.py:109} WARNING - /opt/***/dags/week_3_weather_api_dag_v2.py:73: AirflowProviderDeprecationWarning: The "schema" arg has been renamed to "database" as it contained the database name.Please use "database" to set the database name.
  pg_hook = PostgresHook(

[2025-06-19T13:21:08.035+0000] {base.py:83} INFO - Using connection ID 'my_***_conn' for task execution.
[2025-06-19T13:21:08.041+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-19T13:21:08.049+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=week_3_weather_api_dag_v2, task_id=load_data_to_***, execution_date=20250619T132044, start_date=20250619T132107, end_date=20250619T132108
[2025-06-19T13:21:08.093+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-19T13:21:08.135+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
